lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js

```diff
@@ -120,7 +120,17 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
         
         return courseId;
     },
-            
+
+    _replaceKafUriWithToken: function(sourceUrl, kafUri, kalturaToken) {
+        sourceUrl = this._removeProtocolFromUrl(sourceUrl);
+        kafUri = this._removeProtocolFromUrl(kafUri);
+        return sourceUrl.replace(kafUri, kalturaToken);
```

Comment:
isn't the result here a URL without schema?
if I'm not mistaken - we used to save the schema as part of the embed before this change. not sure it is smart to start storing URLs without schema.